conf/configure-spire.sh

```diff
@@ -57,6 +57,17 @@ for word in ${BOOT_PARAMS}; do
             chmod 600 "${spire_rootdir}/conf/join_token"
         fi
     fi
+
+    if [[ ${parent_key} == 'tpm' ]]; then
+        tpm_setting="${parent_val}"
+
+        if [[ ${tpm_setting} == 'enable' ]]; then
+            echo "Not recreating spire-agent.conf"
+            exit 0
+        elif [[ ${tpm_setting} == 'enroll' ]]; then
+            echo "TPM set to enroll"
+        fi
```

Comment:
Since there's an `elfi`, should there also be an `else` clause just in case `tpm` was set to something other than `enable` or `enroll`?  It could spit out a message like `Unknown TPM setting ${tpm_setting}`